BibBuddy/app/src/main/res/layout/fragment_book_notes.xml

```diff
@@ -1,58 +1,135 @@
 <?xml version="1.0" encoding="utf-8"?>
-
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why change to Relative Layout? Now you cannot scroll to the button.

Reply:
There was an error and I thought it might have been caused by the layout - you are right though, I changed that back.